source/index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+content="widht=device-width
+initial-scale=1.0">
```

Comment:
format your html correctly
```suggestion
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```